### DISCUSSION TITLE
TTS response does not include numeric part of response

### DISCUSSION THREAD

**Discussion by mikey60** [toverainc/willow#233] [Q&A] [answered]
I have an intent that responds with the temperature of the requested area. For example ' Hi ESP basement temperature'.  The ESP Box responds with the correct text response displayed: 'The basement temperature is 71 degrees F.'  The ESP Box audio response is:' The basement temperature is degrees F'.  Is there some configuration setting that would correct this issue?

**Comment by nikito** [comment, marked as answer]
This is a known issue, the TTS presently doesn't handle numbers very well. I think this is something that is going to get more attention post 1.0 launch :)

**Comment by mikey60** [comment]
Thanks for the quick response! I apologize for not searching the issues.  I just searched the discussions.
